frontend/src/pages/Profile.js

```diff
@@ -1,15 +1,303 @@
+
+
+    useEffect(() => {
+
+    fetch("http://localhost:8080/customers/" + currentCustomerId + "/users/username/" + user.username)
+        .then(response => response.json())
+        .then(res => {
+            setUserDto(res);
+            setState(res.addressDto.state);
+        });
+
+
+    }, []);
+
+
+    const editUserHandler = (e) => {
+        e.preventDefault();
+
+        fetch("http://localhost:8080/customers/" +  currentCustomerId + "/users/" + userDto.id, {
+            method: 'PUT',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(userDto)
+        })
+            .then(function (response) {
+                if (response.status !== 200) {
+                    setError('');
+                    setErrors("Something go wrong, try later");
+                } else {
+                    setError('');
+
+                }
+            });
+
+    };
+
+    const editPasswordHandler = (e) => {
+        e.preventDefault();
+
+        fetch("http://localhost:8080/customers/" +  currentCustomerId + "/users/" + userDto.id + "/password", {
+            method: 'PUT',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(userDto.password)
+        })
+            .then(function (response) {
+                if (response.status !== 200) {
+                    setError('');
+                    setErrors("Something go wrong, try later");
+                } else {
+                    setError('');
+
+                }
+            });
+
+    };
+
     return (
         <div style={{
             margin: '70px 10px',
             height: '40px'
         }}>
-            <Alert variant="success">
-                <Alert.Heading>Welcome</Alert.Heading>
+            <Alert variant="success" >
+                <Alert.Heading  >  Info about user </Alert.Heading>
             </Alert>
+            <div className="ProfileCard" style={{
+              alignSelf: "center",
+                padding: "2px 300px 400px 500px"
+            }}>
+
+                <Card style={{width: '25rem'}}
+                      className="shadow p-3 mb-5 bg-white rounded">
+                    <Card.Header className="text-center border-bottom border-primary" style={{'background': 'white'}}>
+                        Info about user
+                    </Card.Header>
+                    <Card.Body>  <Form>
+                        <Form.Group controlId="editUserr" style={{padding: '5px 10px'}}>
+                            <Form.Label>Name</Form.Label>
+                            <Form.Control type="text"
+                                          placeholder="name"
+                                          onChange={handleName}
+                                          value={userDto.name}
+                                          className={
+                                              validError.includes("name")
```

Comment:
This statement is used in many places in this component. We can extract it into separate function with a single param and use it here